sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -59,20 +77,224 @@
 import retrofit2.http.PUT;
 import retrofit2.http.Path;
 import retrofit2.http.Query;
+import retrofit2.http.Tag;
 
 /**
  * PACKAGE PRIVATE CLASS AND METHODS
  */
 final class StorageBlobServiceImpl {
     private final StorageBlobService service;
     private final SerializerAdapter serializerAdapter;
-    private static String XMS_VERSION = "2019-02-02";
+    private static String XMS_VERSION = "2019-12-12";
```

Comment:
Is this the newest Storage version? Where do we check this? Actually, I see different versions than what we had [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/storage/resource-manager/Microsoft.Storage/stable).

Reply:
The link you shared is for the resource manager/management plane. The version update here is for the data plane. It's not quite the latest, but it's the version that enabled tags.